docker/generate_makefile.py

```diff
@@ -68,8 +69,8 @@ def _print_makefile_run_template(image):
         print()
 
 
-# TODO(tanq16): Add unit test.
-def _print_rules_for_image(name, image):
+# TODO(Tanq16): Function must return a string as opposed to printing it.
```

Comment:
Alternatively, what might be both simpler and more useful is to check in the output of this script (ie docker/build.mk), and have a very simple test that compares the output of this script with that file. It'd also allow   verifying the makefile and its changes at review time.

Reply:
To confirm, do the usual write to `docker/generated.mk` and then run the script again and compare output to the file. Is that correct?

Reply:
the test would just run the script and compare it's output to whatever is in docker/generated.mk at the moment

ie. if say a new fuzzer is added, the the change in docker/generated.mk would be explicitly checked in

note that people still wouldn't need to regenerate docker/generated.mk manually, as it automatically gets updated whenever one runs make

Reply:
Okay, I understand. I'll try to write this next CL.